exporter/exportertest/nop_exporter.go

```diff
@@ -45,19 +45,19 @@ func (ne *nopExporter) Shutdown(context.Context) error {
 }
 
 // NewNopTraceExporter creates an TraceExporter that just drops the received data.
-// Deprecated: Use consumertest.NewNopTraces.
+// Deprecated: Use consumertest.NewNopTraceExporter.
```

Comment:
Looks like we some unrelated changes here.

Reply:
yes, true - I noticed this while fixing the rebase conflicts - should this be a seprate PR?

Reply:
Would be better, also the suggestion is wrong

Reply:
ok, dropped the change